### PR TITLE
Switching Privacy forms

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
Right now a user that goes to `/sub-pages/file-a-privacy-complaint/` will face a form about amending a privacy record and a user that goes to `/sub-pages/amending-and-correcting-records-under-the-privacy-act-2/` will see a form to file a privacy complaint. This switches the two forms to fix the issue.

## Review
- @anselmbradford

### CODE REVIEW

File: src/_includes/templates/forms/amending-and-correcting-records-under-the-privacy-act-form.html [hunk 1]
```diff
@@ -141,27 +137,38 @@
                    required>
         </div>
         <div class="form-l_col form-l_col-1">
-            <label class="form-label-header" for="complaint-desc">
-                Description of complaint
+           <label for="request-desc">
+                <span class="form-label-header" for="request-desc">
```
Comment (anselmbradford) [inline src/_includes/templates/forms/amending-and-correcting-records-under-the-privacy-act-form.html]: `for` is not an attribute of `span`.

Comment (anselmbradford) [inline src/_includes/templates/forms/amending-and-correcting-records-under-the-privacy-act-form.html]: Can this span include `display: block` style in some way so as to make the `<br>` that follows it unnecessary?

Comment (KimberlyMunoz) [inline src/_includes/templates/forms/amending-and-correcting-records-under-the-privacy-act-form.html]: Existing bug but totally right to fix.

Comment (anselmbradford) [inline src/_includes/templates/forms/amending-and-correcting-records-under-the-privacy-act-form.html]: Yeah, forgot to mention the for attribute should probably be fixed here, but refactoring to remove the <br> is probably best left to another PR.

Comment (KimberlyMunoz) [inline src/_includes/templates/forms/amending-and-correcting-records-under-the-privacy-act-form.html]: Fixed in f59f6c8

File: src/_includes/templates/forms/file-a-privacy-complaint-form.html [hunk 2]
```diff
@@ -49,7 +53,7 @@
                 </label>
                 <div class="custom-select">
                     <select class="custom-select_select"
-                            data-placeholder="Select State"
+                            data-placeholder="Select state"
```
Comment (anselmbradford) [inline src/_includes/templates/forms/file-a-privacy-complaint-form.html]: Should this be downcased? It's uppercased on https://github.com/cfpb/cfgov-refresh/pull/843/files#diff-1f0688a2a043180c12e1b17d5084cf56R52

Comment (anselmbradford) [inline src/_includes/templates/forms/file-a-privacy-complaint-form.html]: Ahh, nm, that's due to the switching forms looks like.

**Comment by anselmbradford** [comment]
:+1:

**Pull request merged**